### PR TITLE
[1.x] Fix PHP 8.2 compat issue in uses of `DateTime::modify()`

### PR DESCRIPTION
**Pull request by mbabker**
As noted in https://github.com/php/php-src/issues/9950 there is a behavioral change in PHP 8.2 if you make a call like `$dateTime->modify('+-300 seconds')` as a result of another bug fix in the date/time handling libraries.  As the bundle does not enforce TTLs to be a positive number (and the functional tests abuse this a bit), this is a practical fix to ensure anything passing a negative TTL value doesn't catastrophically break.

### CODE REVIEW

**Comment by mbabker** [comment]
Functional CI blocked by #348

**Review by maxhelias — APPROVED** [review]

**Comment by shakaran** [comment]
@maxhelias any update missing for merge this to master? Compat with 8.2 is very important for next upgrades. Thanks

**Comment by maxhelias** [comment]
@shakaran I am not the owner or a collaborator. Thanks

**Comment by mbabker** [comment]
I don't have merge rights here.

**Comment by shakaran** [comment]
> I don't have merge rights here.

who has? By contributions maybe @markitosgv (composer author) or @paulchubatyy ?

**Review by thomaspicquet — APPROVED** [review]

**Comment by mbabker** [comment]
Superseded by #361

**Pull request closed without merging**
